server/api/person/person.email.js

```diff
@@ -12,52 +12,46 @@ module.exports.emailPerson = async (template, to, props, renderOnly = false) =>
   if (to._id && !to.sendEmailNotifications) {
     return null
   }
-  try {
```

Comment:
What impact does removing this try/catch block have?

I see your commit mentions the exception will run up to the request handler - I'm not 100% sure what this will mean for the user but I assume whatever request is in progress will essentially fail. That seems like a large change from the email not sending and the error being logged.

Reply:
I think a failure here implies serious problem with the email subsystem we should be returning 500. We need to make the system intolerant for a while.